### PR TITLE
my first branch

### PR DESCRIPTION
**Pull request by jonwalz**
Another PR! It's best practice NOT to merge your own PRs. 

Why?

So that someone else gets their eyes on your code!

It's often you'll miss something that someone else will see and point it out on here.

Click the little link below that says `my first branch`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # noobspace
 
-What up! Welcome to the world of development. Now add me as a collaborator!
-1. Click the settings tab to far right (after graphs)
-2. Select collaborators
-3. add me!
+Sweet, we don't want people reading this stuff, so I got rid of it.
```
Comment (jonwalz) [inline README.md:3]: Oh I mean to the left. Notice how the text is strike through? Just just indicates what is getting changed.

Comment (jonwalz) [reply README.md:3]: Oops, ha. It's not strikethrough, its red. I just mean we don't need people reading my last instructions.

Comment (pdxhomebrew) [reply README.md:3]: the dashes next to lines 3-6? I noticed your newest line is on 3. Is that replacing the ones in red?

Comment (pdxhomebrew) [reply README.md:3]: as opposed to +?

Comment (pdxhomebrew) [reply README.md:3]: goooootcha.

Comment (jonwalz) [reply README.md:3]: Correct!

Now if you hit the merge button at the bottom, it will replace those.

**Pull request merged**